.travis.yml

```diff
@@ -24,7 +24,7 @@ install:
 - conda config --add channels lightsource2
 - conda create -n testenv pip nose python=$TRAVIS_PYTHON_VERSION pymongo six pyyaml numpy pandas jinja2 boltons prettytable humanize
 - source activate testenv
-- pip install mongoengine
+- 'pip install https://github.com/NSLS-II/doct/zipball/master#egg=doct'
```

Comment:
You can `conda install doct` now. I built master and uploaded it to `lightsource2`